src/common/Finisher.cc

```diff
@@ -3,6 +3,11 @@
 
 #include "Finisher.h"
 
+#include "common/debug.h"
+
+// re-include our assert to clobber the system one; fix dout:
```

Comment:
AFAIK that's correct. I'm not really familiar with our dout()/assert() hackery at this time, other than that I get scads of compiler errors without this. Although "common/debug.h" looks like it should be including "include/assert.h", without including it afterwards, the above source file gives me an error when macro ldout is expanded.